applications/MeshingApplication/custom_processes/mmg_process.cpp

```diff
@@ -822,11 +820,15 @@ void MmgProcess<TMMGLibray>::ExecuteRemeshing()
         }
 
         /* We interpolate the internal variables */
-        InternalVariablesInterpolationProcess InternalVariablesInterpolation = InternalVariablesInterpolationProcess(r_old_model_part, mrThisModelPart, mThisParameters["internal_variables_parameters"]);
-        InternalVariablesInterpolation.Execute();
+        InternalVariablesInterpolationProcess internal_variables_interpolation = InternalVariablesInterpolationProcess(r_old_model_part, mrThisModelPart, mThisParameters["internal_variables_parameters"]);
+        internal_variables_interpolation.Execute();
```

Comment:
Why do you call the assignemnt constructor here?

Reply:
Well, it is just habit, but is not technically wrong

Reply:
Until someone decides to delete the assignment constructor :)
(Happened to me with ModelPart)

Reply:
This is a process, I will not worry about that right now

Reply:
I know, just saying
No need to change it if you know what you are doing